README.md

```diff
@@ -95,7 +95,10 @@ If you want to share Whatsapp and Mailto, you should write `LSApplicationQueries
     ```xml
       <?xml version="1.0" encoding="utf-8"?>
       <paths xmlns:android="http://schemas.android.com/apk/res/android">
+          <-- For files shared using base64 URLs. Required WRITE_EXTERNAL_STORAGE permission -->
           <external-path name="myexternalimages" path="Download/" />
+          <-- For files in internal storage under myInternalDir -->
```

Comment:
This looks good in general, but I think the last bit needed to make sure the "a-ha!" light bulb goes on for people would be to say:

`<-- For files in internal storage "myInternalDir" (replace with your path) -->`

Reply:
nice catch

Reply:
Thanks @mikehardy. I pushed the suggested changes.